pkgs/tools/X11/xpra/default.nix

```diff
@@ -41,6 +41,7 @@ in buildPythonApplication rec {
     (substituteAll {
       src = ./fix-paths.patch;
       inherit (xorg) xkeyboardconfig;
```

Comment:
I think this is not needed anymore?

Reply:
Wait did my answer not post?

The patch references it at line 28

Reply:
I see, I should check whole patch